CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Next
+- `Modal` has new prop `footer`. The footer is sticky by default, but can be turned off with `disableStickyFooter`.
+- `ModalFooter` is added and is used as a wrapper around the content given to `footer`prop.
```

Comment:
`ModalFooter` is not exported, so no need to mention it in the changelog.

```suggestion
```

Reply:
Fixed!